web/themes/custom/pineapple/sass/layouts/nodes/_author.scss

```diff
@@ -0,0 +1,45 @@
+.author-body {
+  .cover-image {
+    height: 350px;
+    margin: -$unit -50% 0 -50%;
```

Comment:
Is there any reason why you didn't use your existing breakout class for this as opposed to rewriting a new line of sass to achieve the same thing?

Reply:
No, I overlooked it sorry. Done.